_posts/2019-10-29-sacred-geometry.md

```diff
@@ -0,0 +1,428 @@
+---
+title:  "Javascript Animation"
+date: 2019-10-29 00:00:00 +0800
+author: jimmy
+---
+
+<div style="width: 85px; float: left; margin-right: 20px;">
+    <img src="/assets/images/articles/tt_small.svg">
+</div>
+
+A quick look at drawing and animation with HTML Canvas and Javascript
+<!--more-->
+
+**TL;DR** Check out this cool javascript animation of multiplication tables <a href='#final_canvas'>below.</a>
```

Comment:
I was just thinking that I should start putting the sexy visuals at the top of my articles, just to draw readers in. I think people are more likely to leave if they don't know how awesome the outcome of the article will be. 

I would be surprised if anyone didn't click this button, sending them to the bottom of the article. Maybe it's worth it to just copy the ending canvas and place it here. What do you think, @jimmykodes?